packages/application-shell/src/utils/http-client.ts

```diff
@@ -172,12 +179,10 @@ const getAppliedForwardToHeaders = (
     [SUPPORTED_HEADERS.X_FORWARD_TO]: forwardToConfig.uri,
     [SUPPORTED_HEADERS.X_FORWARD_TO_AUDIENCE_POLICY]:
       forwardToConfig.audiencePolicy ?? defaultForwardToAudiencePolicy,
-    ...(Boolean(forwardToConfig.exchangeTokenClaims?.length)
+    ...(Boolean(exchangeTokenClaims?.length)
```

Comment:
The condition shouldn't be necessary, because empty headers will be removed by the `omitEmpty`. So this can simply be:

```ts
[SUPPORTED_HEADERS.X_FORWARD_TO_CLAIMS]: exchangeTokenClaims.join(' '),
```

Reply:
Updated in this commit: de46f8f9da017cfa7be43230c0d08b17eaa67e0b